### ISSUE TITLE
Add operability and controllability metrics (RGA, singular values, disturbance gain)

### ISSUE THREAD

**Issue by jkitchin** [jkitchin/differentiable-flowsheets#199]
## Summary

There is no way to ask whether a flowsheet is **controllable**: whether the available manipulated variables have enough independent influence on the controlled variables to reject the disturbances the process will actually see. This is a steady-state, gradient-based question, which makes it a natural fit for a differentiable flowsheet and cheap to compute here relative to anywhere else.

## Evidence that it is absent

```
$ grep -rli "relative gain\|\bRGA\b\|controllab" src/
(only unrelated "condition number" uses in numerics and reconciliation)
```

## What to add

Given a flowsheet, a set of manipulated variables `u`, controlled variables `y`, and disturbances `d`, the steady-state gain matrices `G = dy/du` and `G_d = dy/dd` are one `jax.jacobian` call each. From them:

1. **Relative gain array**, `RGA = G * inv(G).T` elementwise, for input-output pairing and as a warning of strong interaction. Negative diagonal entries flag pairings that are unstable under integral control.
2. **Minimum singular value** of the scaled `G`, as the standard measure of how hard the plant is to control in its worst direction, and the **condition number** as a measure of directionality.
3. **Disturbance condition number** and the disturbance gain, saying whether the available inputs span the directions the disturbances actually push.
4. **Scaling helpers**, since every one of these measures is meaningless unscaled, and getting the scaling right is where users go wrong.

## Why it belongs in difflow

These are gradients of a converged flowsheet with respect to inputs. Computing them by finite differences through a sequential-modular simulator is slow and noisy, which is why controllability screening is usually done on a linear model fitted separately rather than on the design model itself. With AD they are effectively free and exact, and they can therefore be evaluated **inside** a design loop rather than after it.

That last point is the interesting one. It makes it possible to ask whether the economically optimal structure is also a controllable one, which is the central question of integrated design and control and is normally too expensive to pose. `difflow.planning.linearize` and `sensitivity` already compute closely related objects and could likely be reused.

## Proposed implementation

A module `difflow/operability/` exposing `gain_matrix`, `rga`, `min_singular_value`, `condition_number`, `disturbance_gain`, and a `screen(...)` convenience that returns all of them for a candidate design with sensible scaling defaults. Pair it with a short docs page, because the failure mode with these metrics is misuse rather than absence.

## References

Morari, *Chem. Eng. Sci.* 38 (1983) 1881, doi:10.1016/0009-2509(82)80159-0 for the resilience framing. Sakizlis, Perkins and Pistikopoulos, *Comput. Chem. Eng.* 28 (2004) 2069, doi:10.1016/j.compchemeng.2004.03.018 and Yuan et al., *AIChE J.* 58 (2012) 1640, doi:10.1002/aic.13786 for the simultaneous design-and-control literature this would serve.
